### DISCUSSION TITLE
Translation for Link Element in Rich Text Field

### DISCUSSION THREAD

**Discussion by bobbyZ94** [payloadcms/payload#1562] [Feature Requests & Ideas]
The link element pop-up inside the rich text editor is missing translations.

![grafik](https://user-images.githubusercontent.com/63556657/205050811-9e95cc2d-2a3e-49f8-bf2e-05b700ea5c2e.png)

And this little tooltip is also missing translations.

![grafik](https://user-images.githubusercontent.com/63556657/205051209-d51f0cdb-1e67-49ff-94e2-bddbeacf2323.png)

Would it be possible to add these translations or is there a workaround possible with an e.g.: custom slate element?

**Comment by DanRibbens** [comment]
This was fixed in a recent (v1.2.x) release.
